comfy_cli/tracking.py

```
@@ -53,20 +130,38 @@ def disable():
+        try:
+            provider.track(provider_event_name, distinct_id=user_id, properties=dict(properties))
+        except Exception as e:
+            logging.warning(f"Failed to track event via {type(provider).__name__}: {e}")
+
+
+def filter_command_kwargs(kwargs: dict[str, Any]) -> dict[str, Any]:
+    """Drop ``ctx``/``context`` and redact ``SENSITIVE_TRACKING_KEYS`` values."""
+    return {
+        k: ("<redacted>" if v is not None else None) if k in SENSITIVE_TRACKING_KEYS else v
+        for k, v in kwargs.items()
+        if k != "ctx" and k != "context"
```

Pylint (4.0.5)

[refactor] Consider merging these comparisons with 'in' by using 'k not in ('ctx', 'context')'. Use a set instead if elements are hashable.

(R1714)